src/functions/envstore.ts

```diff
@@ -0,0 +1,96 @@
+import { firebaseApiOrigin } from "../api";
+import { FirebaseError } from "../error";
+import { Client } from "../apiv2";
+
+interface EnvStoreEntry {
+  name?: string;
+  vars: Record<string, string>;
+}
+
+const apiClient = new Client({ urlPrefix: firebaseApiOrigin, auth: true });
```

Comment:
initialize with `apiVersion: v1alpha` and remove the prefix in your fetch requests.